pmlc/dialect/affinex/tests/memref_dataflow_opt.mlir

```diff
@@ -0,0 +1,181 @@
+// RUN: pmlc-opt -affinex-memref-dataflow-opt %s | FileCheck %s
+
+// CHECK: simple
+func @simple(%arg: memref<1xf32>) {
+  %cst = constant 0.000000e+00 : f32
+  // CHECK: affine.store
+  affine.store %cst, %arg[0] : memref<1xf32>
+  // CHECK-NOT: affine.load
+  %0 = affine.load %arg[0] : memref<1xf32>
+  // CHECK: addf %cst, %cst
```

Comment:
Generally, we don't want to use raw `%xyz` in CHECK statements. 
https://mlir.llvm.org/getting_started/TestingGuide/#filecheck-best-practices

Reply:
Done.